index.html

```diff
@@ -182,14 +182,18 @@ <h3 id="unacceptablebehaviour">
         <li>Retaliating against anyone who files a complaint that someone has violated this code of conduct.</li>
       </ul>
 
-      <p>The enforcers of this Code should prioritise the safety of individuals within marginalised communities over the comfort of others,
-        and reserve the right not to act on complaints regarding:</p>
+      <p>
+	All complaints will be taken seriously and will recieve a response, 
+	but enforcers of this Code will prioritise the safety of individuals
```

Comment:
I am okay with this new language in general, however I don't like the overly prescriptive tone of it with the change from "should" to "will" (which is a departure from its original form). Suggest to revert back to should, i.e. "the enforcers of this Code should prioritise ..."

Reply:
I'm leaning toward preferring "... will be taken seriously and _should_ receive a response, but enforcers of this code will prioritise the safety of individuals..."

Reply:
I don't think it's fair not to give any kind of response to a complaint, ever, because it suggests the complaint was not taken seriously. If we get a DOS scenario swamping the Ombuds with spurious claims, that's something we should tackle at the time; I don't think it's something we should plan ahead for.

Reply:
Indeed, I was thinking about the DOS scenario.  I agree in general that complaints shouldn't be ignored

Reply:
"will" is now restored to "should" the all complaints will be taken seriously has been moved to the top of the document.